pdfstitcher.py

```diff
@@ -747,6 +749,13 @@ def load_file(self,pathname):
     h = min(int(disp_h*0.85),800)
     w = min(int(disp_w*0.60),700)
 
+    if sys.platform.startswith('win32'):
+        ctypes.windll.shcore.SetProcessDpiAwareness(True)
+        user32 = ctypes.windll.user32
+        screensize = user32.GetSystemMetrics(0), user32.GetSystemMetrics(1)
+        h = int(screensize[1] * 0.85)
```

Comment:
The window did not draw nicely on 4k monitor, all components were munched together, that's why i left out the min() with 800 and 700 values and let the window to adjust just dynamically.